### PR TITLE
#91 Fix for token refresh expiration delta

### PR DESCRIPTION
**Pull request by themotu**
updated to accurately match documentation making `when: t = orig_iat + JWT_REFRESH_EXPIRATION_DELTA` work as described

### CODE REVIEW

**Comment by themotu** [comment]
Need to make an additional field to the token for this to work properly without a database or redis connection

**Comment by cwirz** [comment]
@themotu any updates?

File: graphql_jwt/utils.py [hunk 1]
```diff
@@ -41,11 +41,13 @@ def jwt_encode(payload, context=None):
     ).decode('utf-8')
 
 
-def jwt_decode(token, context=None):
+def jwt_decode(token, context=None, verify=None):
+    if verify == None:
```
Comment (cwirz) [inline graphql_jwt/utils.py]: Please change to 
`if verify is None:`

**Comment by mongkok** [comment]
hey @themotu , 
What's the bug?

**Comment by mongkok** [comment]
This PR allows a single use token to be refreshed once has expired `orig_iat + jwt_settings.JWT_EXPIRATION_DELTA`.
I close the PR, please let me know if you have any questions.

**Pull request closed without merging**